AUTOTEST/machine-lassen.sh

```diff
@@ -46,7 +46,7 @@ eo=""
 # CUDA with UM in debug mode [ij, ams, struct, sstruct]
 co="--with-cuda --enable-unified-memory --enable-persistent --enable-debug --with-cuda-arch=\\'60 70\\' --with-extra-CFLAGS=\\'-qmaxmem=-1 -qsuppress=1500-029\\' --with-extra-CXXFLAGS=\\'-qmaxmem=-1 -qsuppress=1500-029\\'"
 ro="-ij-gpu -ams -struct -sstruct -rt -mpibind -save cuda"
-./test.sh basic.sh $src_dir -co: $co -mo: $mo -ro: $ro
+./test.sh basic.sh $src_dir -co: $co -mo: $mo -ro: $ro -eo: -gpu
```

Comment:
@rfalgout Just to confirm I need to have `-rt -mpibind -save cuda` in `-eo` even if I have them in `-ro`? Thanks again!

Reply:
Yes.  The `-co:`, `-mo:`, `-ro:`, and `-eo:` options for `basic.sh` are completely separate from each other.  The information to put in each of them is passed directly to the `configure.sh`, `make.sh`, `run.sh`, and `examples.sh` tests.

Reply:
Great! Thanks for explaining. I will make the change.